scripts/dts/extract_dts_includes.py

```diff
@@ -443,7 +445,12 @@ def max_dict_key(dct):
 
         for alias in sorted(defs[node]['aliases']):
             alias_target = defs[node]['aliases'][alias]
-            f.write(define_str(alias, alias_target, value_tabs))
+            deprecated_warn = False
+            # Mark any non-DT_ prefixed define as deprecated except
+            # for now we special case LED, SW, and *PWM_LED*
+            if not alias.startswith(('DT_', 'LED', 'SW')) and not 'PWM_LED' in alias:
+                deprecated_warn = True
```

Comment:
Maybe

```python
deprecated_warn = not alias.startswith(('DT_', 'LED', 'SW')) and not 'PWM_LED' in alias
```

or

```python
f.write(define_str(
    alias, alias_target,
    # Mark any non-DT_-prefixed define...
    not alias.startswith(('DT_', 'LED', 'SW')) and
        not 'PWM_LED' in alias))
```

TBH I've probably overdone the 80-column limit thing in the past too. Not the end of the world if a few things go over it...